presto-main/src/main/resources/webapp/query.html

```diff
@@ -297,7 +297,20 @@
 }
 
 function formatCumulativeMemory(cumulativeMemory) {
-    return (cumulativeMemory / Math.pow(1000.0, 4)).toLocaleString() + 'GB seconds';
+    unit = 'GB seconds';
```

Comment:
There's a formatDataSize function available. Use that instead of rolling your own here.